scripts/popup.js

```diff
@@ -154,7 +157,32 @@
     });
   }
 
+  function suggestNewFilename(item, suggest) {
+    var newFilename = '';
```

Comment:
Renamed `new_filename` to `newFilename` - variables use `camelCase`, except for `localStorage` properties and JQuery elements, which use `snake_case`.